WriterDesk1/src/setupProxy.js

```diff
@@ -4,7 +4,7 @@ module.exports = function(app) {
   app.use(
     '/api',
     createProxyMiddleware({
-      target: 'https://localhost:5000',
+      target: 'https://backend-lv2fs4hn6a-ew.a.run.app',
```

Comment:
Ik neem aan dat dit gekke ding ook goed is?

Reply:
Dit is useless ondertussen